code/modules/mob/living/carbon/human/life.dm

```diff
@@ -808,7 +808,8 @@
 			var/total_phoronloss = 0
 			for(var/obj/item/I in src)
 				if(I.contaminated)
-					total_phoronloss += vsc.plc.CONTAMINATION_LOSS
+					if(src.species && src.species.get_bodytype() != "Vox")

```

Comment:
This should be tweaked if we're going back to Nitrogen being good and Phoron being bad.